functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -147,7 +149,7 @@ def get_user(conn, user_id, _redis):
 
 def get_user_cache(_redis):
```

Comment:
in order to handle CACHE_TOGGLE you should wrap the body like this in an if statement =>
```
    if (CACHE_TOGGLE == "On"):
        try:
            cache = _redis.get(ALL_USERS_KEY)
            return cache
        except TypeError as e:
            logging.critical('Failed to fetch user from cache: ' + e.args[1])
            return None
    else:
        return None
```